__tests__/html/offlineUI.fatalError.html

```diff
@@ -15,7 +15,7 @@
   <body>
     <div id="webchat"></div>
     <script crossorigin="anonymous" src="/__dist__/webchat-es5.js"></script>
-    <script type="text/babel" data-presets="es2015,stage-3">
+    <script type="text/babel" data-presets="env,stage-3">
```

Comment:
Why `stage-3`? deprecated. For IE11?

Reply:
`stage-3` is something not in "production" yet, but very close.

`es2015` is deprecated and also targeting ES6. Everyone should use `env` instead, which by default is minus 2 versions including IE11.